float32[] dlib_val
float32[] max_ref
float32 distX
float32 distY
float32 distW
float32 distH
